modules/core/src/main/java/org/apache/fluo/core/client/FluoAdminImpl.java

```diff
@@ -516,6 +516,22 @@ public boolean oracleExists() {
     return oracleExists(getAppCurator());
   }
 
+  public static int numOracles(CuratorFramework curator) {
+    int numOracles = 0;
+    if (oracleExists(curator)) {
+      try {
+        numOracles += curator.getChildren().forPath(ZookeeperPath.ORACLE_SERVER).size();
```

Comment:
I'm a little concerned that the LeaderLatch may create some Ephemeral ZNodes during leader election that would cause this to be temporarily inaccurate.

Reply:
This might not actually be the best idea because it relies on the Curator LeaderLatch implementation.
The docs seem to advise against using the paths given to Curator Recipes: https://cwiki.apache.org/confluence/display/CURATOR/TN7

Reply:
Never mind I'm pretty sure I found a much better way in the new commit.